diff --git a/media/libstagefright/OMXCodec.cpp b/media/libstagefright/OMXCodec.cpp
index 27f7120..f2850bc 100644
--- a/media/libstagefright/OMXCodec.cpp
+++ b/media/libstagefright/OMXCodec.cpp
@@ -1831,7 +1831,7 @@ status_t OMXCodec::allocateOutputBuffersFromNativeWindow() {
     err = mNativeWindow->query(
                 mNativeWindow.get(), NATIVE_WINDOW_QUEUES_TO_WINDOW_COMPOSER,
                 &is_native_window);
-
+#if 0
     if(err==0)
     {
         /* Only videos which are shown in native windows can be rescaled */
@@ -1905,6 +1905,7 @@ status_t OMXCodec::allocateOutputBuffersFromNativeWindow() {
             }
         }
     }
+#endif
 
     err = native_window_set_buffers_geometry(
             mNativeWindow.get(),
